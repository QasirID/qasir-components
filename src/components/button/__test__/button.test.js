// import React from 'react';
// import PropTypes from 'prop-types';
// import renderer from 'react-test-renderer';
// import { mount, shallow } from 'enzyme';

// import Button from '..';

// describe('Button component', () => {
//     it('Should exist', () => {
//         expect(Button).toMatchSnapshot();
//     });
//     it('Should be a default Button', () => {
//         const component = renderer.create(<Button />);
//         expect(component.toJSON()).toMatchSnapshot();
//     });
// });