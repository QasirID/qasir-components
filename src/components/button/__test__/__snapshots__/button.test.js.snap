// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component Should be a default Button 1`] = `
<button
  className="sc-bdVaJa fQAHqL"
  size="md"
  type="button"
/>
`;

exports[`Button component Should exist 1`] = `[Function]`;
